provider/datagen/src/transform/cldr/time_zones/convert.rs

```diff
@@ -444,47 +446,56 @@ fn metazone_periods_iter(
         Vec<MetaZoneForPeriod>,
         LiteMap<String, MetaZoneId>,
     ),
-) -> impl Iterator<Item = (TimeZoneBcp47Id, String, Option<MetaZoneId>)> {
+) -> impl Iterator<Item = (TimeZoneBcp47Id, i32, Option<MetaZoneId>)> {
     let (time_zone_key, periods, meta_zone_id_data) = pair;
     periods
         .into_iter()
         .map(move |period| match &period.uses_meta_zone.from {
             Some(from) => {
+                let parts: Vec<String> = from.split(' ').map(|s| s.to_string()).collect();
```

Comment:
Thought: This is fine for now, but ideally this parsing can move into a library function. Filed https://github.com/unicode-org/icu4x/issues/2127 for that.

Reply:
I will add a TODO here